packages/orchestrator-ui-components/src/rtk/endpoints/processSteps.ts

```diff
@@ -0,0 +1,51 @@
+import { TimelineItem } from '@/components';
+import {
+    Process,
+    ProcessStepsResult,
+    StartProcessStep,
+    StepStatus,
+} from '@/types';
+
+import { orchestratorApi } from '../api';
+
+export const processStepsQuery = `
+    query ProcessSteps($processName: String!) {
+        workflows(filterBy: { field: "name", value: $processName }) {
+            page {
+                steps {
+                    name
+                    assignee
+                }
+            }
+        }
+    }
+`;
+
+const processStepApi = orchestratorApi.injectEndpoints({
+    endpoints: (build) => ({
+        getTimeLineItems: build.query<TimelineItem[], Process['workflowName']>({
+            query: (processName) => ({
+                document: processStepsQuery,
+                variables: { processName },
+            }),
+            transformResponse: (
+                response: ProcessStepsResult,
+            ): TimelineItem[] => {
+                const timeLineItems: TimelineItem[] = response.workflows
```

Comment:
According to the type definition the workflows prop (ProcessStepsResult) is always available, there for there is no need for using `?` here. Same for page[0].

Reply:
Fixed